ivy/functional/frontends/torch/linalg.py

```diff
@@ -80,6 +80,11 @@ def matrix_power(input, n, *, out=None):
     return ivy.matrix_power(input, n, out=out)
 
 
+@to_ivy_arrays_and_back
+def matrix_exp(input, *, out=None):
+    return torch_frontend.matrix_power(input, out=out)
```

Comment:
We should also implement the test for the frontend function `matrix_exp`.

Reply:
sorry for the late response, I'll be working on this today :)